src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3623,7 +3610,7 @@ void InitSolarHeatGains(EnergyPlusData &state)
         if (state.dataSurface->UseRepresentativeSurfaceCalculations) {
             for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
                 for (int spaceNum : state.dataHeatBal->Zone(zoneNum).spaceIndexes) {
-                    auto &thisSpace = state.dataHeatBal->space(spaceNum);
+                    auto const &thisSpace = state.dataHeatBal->space(spaceNum);
```

Comment:
Lots of these:

[src/EnergyPlus/HeatBalanceSurfaceManager.cc:3626]:(style),[constVariable],Variable 'thisSpace' can be declared as reference to const